README.md

```diff
@@ -33,3 +33,21 @@ In the pull request, please make it clear how we can verify that the contributio
 This could be accomplished with a link to a demonstration experiment, the inclusion of an example file and/or testing files, or through some other means.
 We try to review pull requests quickly and add new contributions as soon as the minimal standards are met.
 
+## Plugin templates
+
+There are two plugin template directories inside the `/packages` directory that you can as a reference when creating a directory for your plugin contribution. Both templates are compatible with jsPsych v7+.
+
+* `plugin-template-ts`: This template uses TypeScript source files that are complied into JavaScript using Node.js and npm. This is the format used for plugins in the main jsPsych repo. For more details, please see the jsPsych documentation page [Configuring the jsPsych development environment](https://www.jspsych.org/developers/configuration).
```

Comment:
```suggestion
* `plugin-template-ts`: This template uses TypeScript source files that are complied into JavaScript using Node.js and npm.
  This is the format used for plugins in the main jsPsych repo.
  For more details, including setup instructions, please see the jsPsych documentation page [Configuring the jsPsych development environment](https://www.jspsych.org/latest/developers/configuration).
```
@jodeleeuw Is this enough info for the NPM stuff here? I'll add a warning box about the `npm install` location in the config docs anyway.

Reply:
Yeah this works for me!